docs/config.md

```diff
@@ -227,6 +227,9 @@ cpu_only_composing_models: <comma-delimited-string-list>
 # Enables the searching of request rate (instead of concurrency)
 [ request_rate_search_enable: <bool> | default: false]
 
+# Always report GPU metrics
+[ always_report_gpu_metrics: <bool> | default: false]
```

Comment:
I don't love the long name. Looking around, I see "collect-cpu-metrics". Can we parallel that with "collect-gpu-metrics"? And state that the default depends on cpu-only of the model(s)?

I guess that might be messy though because then you need to know if it was specified by the user or not?

Reply:
I don't mind the name (obviously!). I much prefer it to be slightly longer and descriptive over the alternative.